From fa6ce1fa81dfde47d57ea111931de106e2977ebc Mon Sep 17 00:00:00 2001
From: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
Date: Wed, 4 Oct 2017 09:55:58 +0200
Subject: [PATCH 056/437] mfd: syscon: atmel-smc: include string.h

The string.h header file is needed for the memset() definition. The RT
build fails because it is not pulled in via other header files.

Signed-off-by: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
---
 drivers/mfd/atmel-smc.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/mfd/atmel-smc.c b/drivers/mfd/atmel-smc.c
index 7d77948567d7..0adbd2e796fe 100644
--- a/drivers/mfd/atmel-smc.c
+++ b/drivers/mfd/atmel-smc.c
@@ -12,6 +12,7 @@
  */
 
 #include <linux/mfd/syscon/atmel-smc.h>
+#include <linux/string.h>
 
 /**
  * atmel_smc_cs_conf_init - initialize a SMC CS conf
-- 
2.19.1

